src/app/src/components/DrawTools/DrawTools.js

```diff
@@ -45,6 +76,18 @@ export default function DrawTools({ details }) {
     );
 }
 
+function getDrawMode({ status, userRole }) {
+    if (userRole === ROLES.VALIDATOR && status === BOUNDARY_STATUS.IN_REVIEW) {
```

Comment:
This used to be:
```js
        [BOUNDARY_STATUS.SUBMITTED, BOUNDARY_STATUS.IN_REVIEW].includes(
            details.status
        ) &&
        user.role === ROLES.VALIDATOR
```
I'm thinking that the distinction between `IN_REVIEW` and `SUBMITTED` is the presence of a row in the Reviews database table. If a Validator clicks "Review" on a boudary, an entry should be added to the Reviews table before taking them to the `/draw` page. 

Otherwise the Review creation logic would happen the first time a review is saved. This feels more complicated to me, however, it's something that could be handled entirely by the back-end.

I'm thinking that the distinction between `IN_REVIEW` and `SUBMITTED` is the presence of a row in the Reviews database table

Reply:
This understanding is correct.

DRAFT = Row in Submission table, with submitted_at = NULL
SUBMITTED = That row has a value for submitted_at
IN_REVIEW = Row in Review table, with reviewed_at = NULL
NEEDS REVISIONS = That row has a value for reviewed_at
APPROVED = Row in Approval table, with a value for approved_at

I'm not sure how much we'll end up using IN_REVIEW in practice.